bee-ternary/src/lib.rs

```diff
@@ -719,8 +719,8 @@ impl<T: RawEncoding + ?Sized> fmt::Display for Trits<T> {
 
 /// A buffer containing trits.
 ///
-/// This type is roughly analogous to [`Vec`] or [`String`]. It supports pushing and popping trits
-/// and dereferences to [`Trits`]. It may be borrowed as a trit slice, either mutably or immutably.
+/// This type is roughly analogous to [`alloc::vec::Vec`] or [`alloc::string::String`]. It supports pushing and popping
```

Comment:
Can you use the version with parentheses instead please ? Otherwise that's the actual text showing.
Like ```[`Vec`](alloc::vec::Vec)```